### DISCUSSION TITLE
Add a flag to indicate authenticated ORCID iD

### DISCUSSION THREAD

**Discussion by hana-her** [datacite/datacite-suggestions#140] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

Repositories are instructed to collect authenticated ORCID iDs (https://info.orcid.org/documentation/collecting-and-sharing-orcid-ids/). When they strive to do that and implement a mechanism for researchers to authenticate their ORCID iDs, that information gets lost once again in DataCite DOIs associated with the repository's content. Adding the flag to ORCID iDs to indicated whether it was authenticated by their owners would add additional layer of trust to DataCite metadata.

### What solution might meet your needs?

Adding a field to indicate whether an ORCID iD has or has not been authenticated

### Your name

Hana Heringová

### Your organization

National Library of Technology

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [X] Developers building integrations with DataCite services
- [X] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by hana-her** [comment]
Hello there, just following up on this suggestion.

We (National Library of Technology, Prague) had a chance to very briefly disscuss this with ORCID - apparently, this flag was implemented in Crossref metadata schema but it might not be as functional as expected as it is being misused and people are flagging unauthenticated ORCID iDs manually as authenticated.

**Comment by codycooperross** [comment]
Thank you for this update, @hana-her ! Do you have or did ORCID have suggestions for implementing authenticated ORCIDs in a different way?

**Comment by hana-her** [comment]
Unfortunately, not that I know of, I believe it was something to look into at this point @codycooperross
